.github/workflows/ci-examples-app.yml

```diff
@@ -12,6 +12,7 @@ on:
       - ".github/workflows/ci-examples-app.yml"
       - "src/lightning_app/**"
       - "tests/integrations_app/**"
+      - "!tests/integrations_app/flagship/**"
```

Comment:
@Borda I added these exclusions to match the checkgroup config

Reply:
Yes, good ones, thx 🐰